components/common-go/baseserver/server.go

```diff
@@ -387,3 +396,14 @@ func (s *builtinServices) Close() error {
 	eg.Go(func() error { return s.Health.Close() })
 	return eg.Wait()
 }
+
+func httpAddress(cfg *ServerConfiguration, l net.Listener) string {
+	if l == nil {
+		return ""
+	}
+	protocol := "http"
+	if cfg != nil && cfg.TLS != nil {
+		protocol = "https"
+	}
+	return fmt.Sprintf("%s://%s", protocol, l.Addr().String())
+}
```

Comment:
It's not related to this PR, please rebase to the latest main branch.

Reply:
It is, this is just a move of the function to the bottom to not be in the middle of instance methods of the Server. The github diff renders funny in this case.

Reply:
> The github diff renders funny in this case.

How can we trust the GitHub diff 😆